engines/account/app/mailers/account/signup_mailer.rb

```diff
@@ -13,11 +13,9 @@ def confirm_email(signup)
 
     private
 
-    # TODO Find a way to manage express URLs
-    #
     # @return [String]
     def confirm_url
-      "http://10.0.3.70:8000/signup/complete/#{@signup.token}"
+      "#{::Shared::PublicUrl.base_url}/signup/complete/#{@signup.token}"
```

Comment:
I thought about that, but for now we'll build each of them only in one place. When we'll find ourselves duplicating the URLs in different places we will create a `FrontendUrl` generator for each engine.